src/rlx_tar.erl

```diff
@@ -62,7 +54,7 @@ make_tar(State, Release, OutputDir) ->
 
 update_tar(State, TempDir, OutputDir, Name, Vsn, ErtsVersion) ->
     IncludeErts = rlx_state:get(State, include_erts, true),
-    SystemLibs = rlx_state:get(State, system_libs, true),
+    SystemLibs = rlx_state:get(State, include_system_libs, IncludeErts),
```

Comment:
does this represent a breakage in config format we'll have to correct back in rebar3 to maintain compat?

Reply:
agh, yes, changing it back. I was going to move it to an actual value and not just something in a dictionary but in case I forget to do that I'll set it back to `system_libs` for now.